AUTHORS.md

```diff
@@ -28,7 +35,8 @@ Garry Vaughan, William Weibel, and many others, including their advice on how
 to make GMT portable to a wide range of platforms. John Lillibridge and Stephan
 Eickschen provided the original Examples (11) and (32), respectively; Hanno von
 Lom helped resolve early problems with DLL libraries for Win32; Lloyd Parkes
-enabled indexed color images in PostScript; Kurt Schwehr maintains the fink packages;
+enabled indexed color images in PostScript; Brent Wood developed the GMT/OGR format;
+Kurt Schwehr maintains the fink packages;
```

Comment:
Hmm, but we don't acknowledge the other package maintainers!

Reply:
Feel free to expand on this file!

Reply:
Well, what I meant is that acknowledging package managers is a bit beyond the purpose of this list.

Reply:
That is true, agree we can remove that.